A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -1453,23 +1474,52 @@ var A320_Neo_UpperECAM;
         getActiveFailures() {
             var output = {};
             this.hasActiveFailures = false;
+            this.hasWarnings = false;
+            this.hasCautions = false;
             for (var i = 0; i < this.messages.failures.length; i++) {
```

Comment:
Should be `let`.

Reply:
@Benjozork tyler didn't touch that....

Reply:
actually, maybe i cant tell what line you're talking about